README.md

```diff
@@ -124,13 +124,16 @@ _The IP address of the machine is not returned yet as this requires a call to a
 
 Create a new machine by cloning an existing machine or a template. This machine will be bootstrapped by Chef, as long as all the relevant details are in the `knife.rb` file.
 
-Parameters that are required are:
+The following parameters are required:
 
- - `--targethost` - The host that the virtual machine should be created on
- - `--folder` - Folder that machine should be stored in. This must already exist.
  - `--datacenter` - Datacenter in the vSphere environment that controls the target host
  - `--template` - Name of the virtual machine or template to use
 
+There are some optional parameters that can be specified:
+
+ - `--targethost` - The host that the virtual machine should be created on. If not specified the first host in the cluster is used.
+ - `--folder` - Folder that machine should be stored in. If specified this must already exist.
+
```

Comment:
We should add the `--pool` option here. With a note on with the caveat that it finds the first resource pool if one isn't declared.